cfme/tests/configure/test_access_control.py

```diff
@@ -161,6 +161,19 @@ def test_user_remove_tag():
     user.delete()
 
 
+def test_current_user_login_delete():
+    group_user = ac.Group("EvmGroup-super_administrator")
+    user = ac.User(
+        name='user' + fauxfactory.gen_alphanumeric(),
+        credential=new_credential(),
+        email='xyz@redhat.com',
+        group=group_user)
+    user.create()
+    login.login(user.credential.principal, user.credential.secret)
```

Comment:
I believe the user.delete finalizer should belong after user.create and login.login_admin before login.login.